EdFi.Buzz.UI.Angular/src/app/Features/SurveyAnalyticsReact/surveyAnalyticsReactWrapper.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnDestroy,
+  Output,
+  SimpleChanges,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import * as React from 'react';
+
+import * as ReactDOM from 'react-dom';
+import { ApiService } from 'src/app/Services/api.service';
+import { SurveyAnalytics } from './surveyAnalytics';
+
+const containerElementName = 'surveyAnalyticsReactComponentContainer';
+
+@Component({
+  selector: 'app-survey-analytics-react',
+  template: `<span #${containerElementName}></span>`,
+  styleUrls: ['./surveyAnalytics.css', './surveyMetadataUI.css', '../../Components/DataTable/dataTable.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class SurveyAnalyticsReactWrapperComponent implements OnChanges, OnDestroy, AfterViewInit {
+  @ViewChild(containerElementName, {static: false}) containerRef: ElementRef;
+
+  @Input() public counter = 100;
```

Comment:
Can we remove the counter? Is that still used?

Reply:
Yes, its removed